backend/entities/AccessRequest.cs

```diff
@@ -1,92 +1,32 @@
 using System;
 using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
 
 namespace Pims.Dal.Entities
 {
     /// <summary>
     /// AccessRequest class, provides an entity for the datamodel to manage submitted access request forms for unauthorized users.
     /// </summary>
-    [MotiTable("PIMS_ACCESS_REQUEST", "ACRQST")]
-    public class AccessRequest : BaseAppEntity
+    public partial class PimsAccessRequest : IBaseAppEntity
```

Comment:
abstract base class no longer works as duplicated fields (ie all the App*, DB* and concurrency fields part of every table) would need to override those fields post generation.